test/index.test.ts

```diff
@@ -366,3 +366,12 @@ test('throw error in beforeAll, afterAll, beforeEach, afterEach', async () => {
   expect(bench.getTask('AE test')!.result!.error).toBe(AEerror);
   expect(bench.getTask('AA test')!.result!.error).toBe(AAerror);
 });
+
+test('remove non-existent task', () => {
+  const bench = new Bench();
+  bench.addEventListener('remove', () => {
+    throw new Error('should not be called');
+  });
+
+  bench.remove('non-existent');
```

Comment:
check how vitest tests for non throwing.

Reply:
No errors, just an empty event received in the listener

Reply:
There is `expect.unreachable()` for situations like this

Reply:
Approval required to update version for "vitest"

Reply:
Update to latest vitest if necestt